### DISCUSSION TITLE
@Mike777Khoo 支付宝购买专业版问题

### DISCUSSION THREAD

**Discussion by taoistx** [getlantern/lantern-forum-cn#394] [General]
@Mike777Khoo 我是通过支付宝的方式购买的专业版，没有激活码，删掉global.yaml无法恢复专业版。官方软件报错渠道反馈是重置账号密码，但中文版蓝灯根本没有输入密码的地方。希望得到有效的回复

**Comment by MaxMKMax** [comment]
@taoistx 
你好。您可以向我提供您在专业版中注册的电子邮件吗？还有付款收据。我会帮助您与团队核实。如果我们方面没有进一步回复，请在 github 上私信我相同的链接。我再帮你跟进一下。谢谢。

**Reply by taoistx** [reply]
![微信图片_20240404213913](https://github.com/getlantern/lantern-forum-cn/assets/60421344/c19b9966-a355-4574-8a23-f89f9fd41dc5)

注册邮箱为 [taoistx@163.com](mailto:taoistx@163.com)

**Comment by taoistx** [comment]
@Mike777Khoo  有进展吗？

**Comment by taoistx** [comment]
@Mike777Khoo 希望能尽快把验证码发到我邮箱，谢谢

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/6014049a-a024-4520-b4cb-5a113dd715fe)

@taoistx 您好. 11 天前给您上报情况过了. 客服也在 7 天前给您发邮件了. 里面有激活码. 请您查看邮箱.
